src/Network/Minio.hs

```diff
@@ -88,6 +88,7 @@ module Network.Minio
 
   -- ** Conduit-based streaming operations
   , putObject
+  , GetObjectOptions(..)
```

Comment:
This should be `GetObjectOptions` so that the constructor is not exported.

Reply:
Fixed